### PR TITLE
Inquiry service

### PR DESCRIPTION
**Pull request by keshavgr8**
- [x] Pivotal Story #154051043 Inquiry Register UI integration
- [x] Pivotal Story #154051132 Error notification grid setup

### CODE REVIEW

File: src/pages/inq-details/inq-details.html [hunk 1]
```diff
@@ -0,0 +1,327 @@
+<!--
+  Generated template for the InqDetailsPage page.
+
+  See http://ionicframework.com/docs/components/#navigation for more info on
+  Ionic pages and navigation.
+-->
```
Comment (aayushKhandpur) [inline src/pages/inq-details/inq-details.html]: Remove auto-generated comments

File: src/pages/inq-details/inq-details.html [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+              <ion-input type="email" formControlName="email"></ion-input>
+            </ion-item>
+          </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col>
+            <ion-item>
+              <ion-label>Gender</ion-label>
+              <ion-select formControlName="gender">
+                <ion-option value="male">Male</ion-option>
```
Comment (aayushKhandpur) [inline src/pages/inq-details/inq-details.html]: Should use enum value here instead of hardcoding

File: src/pages/inq-details/inq-details.html [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+                <ion-option value="female">Female</ion-option>
+                <ion-option value="other">Other</ion-option>
+              </ion-select>
+            </ion-item>
+          </ion-col>
+          <ion-col>
+            <ion-item>
+              <ion-label>Date of Birth</ion-label>
+              <ion-datetime displayFormat="DD/MM/YYYY" pickerFormat="DD/MM/YYYY" formControlName="dob"></ion-datetime>
```
Comment (aayushKhandpur) [inline src/pages/inq-details/inq-details.html]: Can we add a check and not allow the user to pick a future date?

File: src/pages/inq-details/inq-details.html [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+            </ion-item>
+          </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col>
+            <ion-item>
+              <ion-label>Education</ion-label>
+              <ion-select formControlName="highestEducation">
+                <ion-option value="ssc">SSC</ion-option>
```
Comment (aayushKhandpur) [inline src/pages/inq-details/inq-details.html]: Use Enum value or JS array and iterate. Should be sorted.

File: src/pages/inq-details/inq-details.html [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+                <ion-option value="hsc">HSC</ion-option>
+                <ion-option value="undergraduate">Under Graduate</ion-option>
+                <ion-option value="graduate">Graduate</ion-option>
+                <ion-option value="postgraduate">Post Graduate</ion-option>
+              </ion-select>
+            </ion-item>
+          </ion-col>
+          <ion-col>
+            <ion-item>
+              <ion-label>Computer Knowledge</ion-label>
```
Comment (aayushKhandpur) [inline src/pages/inq-details/inq-details.html:64]: Use enum or JS array

File: src/pages/inq-details/inq-details.html (hunk 3 above)
Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html]: Yes we can, what should be the upper and lower limits?

File: src/pages/inq-details/inq-details.html (hunk 1 above)
Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html]: resolved in commit f476f94

File: src/pages/inq-details/inq-details.html (hunk 2 above)
Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html]: Resolved in commit 8ef72d4

File: src/pages/inq-details/inq-details.html (hunk 4 above)
Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html]: Resolved in commit 8ef72d4

File: src/pages/inq-details/inq-details.html (hunk 5 above)
Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html:64]: Resolved in commit 8ef72d4

File: src/pages/inq-details/inq-details.html (hunk 3 above)
Comment (aayushKhandpur) [reply src/pages/inq-details/inq-details.html]: Lower limit can be avoided(or use any logical limit if it is mandatory. Maybe 1900) Upper limit should not be greater than today's date

Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html]: Resolved in commit 62215b7

Comment (keshavgr8) [reply src/pages/inq-details/inq-details.html]: And in commit 2bfb92f

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**